helm/nessie/values.yaml

```
@@ -552,6 +552,15 @@ tracing:
     {}
     # service.name: my-nessie
 
+metrics:
+  # -- Specifies whether metrics for the nessie server should be enabled.
+  enabled: true
```

```suggestion
  enabled: false
```
(because there are no tags by default?)

---

In application.properties, it's enabled by default. Do we want to change that for the Helm chart?

---

Hmm also it seems we can't really disable metrics:

```
Caused by: jakarta.enterprise.inject.UnsatisfiedResolutionException: Unsatisfied dependency for type io.micrometer.core.instrument.MeterRegistry and qualifiers [@Default]
        	- injection target: parameter 'meterRegistry' of org.projectnessie.quarkus.providers.storage.PersistProvider#produceCacheBackend()
        	- declared on PRODUCER METHOD bean [types=[org.projectnessie.versioned.storage.cache.CacheBackend, java.lang.Object], qualifiers=[@Default, @Any], target=org.projectnessie.versioned.storage.cache.CacheBackend produceCacheBackend(io.micrometer.core.instrument.MeterRegistry meterRegistry, org.projectnessie.versioned.storage.cache.DistributedCacheInvalidation invalidationSender, org.projectnessie.quarkus.providers.storage.CacheInvalidationReceiver cacheInvalidationReceiver), declaringBean=org.projectnessie.quarkus.providers.storage.PersistProvider]
        	at io.quarkus.arc.processor.Beans.resolveInjectionPoint(Beans.java:519)
        	at io.quarkus.arc.processor.BeanInfo.init(BeanInfo.java:642)
        	at io.quarkus.arc.processor.BeanDeployment.init(BeanDeployment.java:317)
        	... 14 more
```

---

Ok that's fixed, but I think disabling metrics by default would be a breaking change.

---

Leaving metrics always enables works for me. I don't see a need to turn it off at all.